config.json

```diff
@@ -1378,6 +1378,468 @@
+        "slug": "channels",
+        "name": "Channels",
+        "blurb": ""
+    },
+    {
+        "uuid": "5c61a070-0585-4e23-8620-46c7c73f2fe1",
+        "slug": "char",
+        "name": "Char",
+        "blurb": ""
+    },
+    {
+        "uuid": "1cbd4801-6c66-4c56-902d-b9c5a0093915",
+        "slug": "collect",
+        "name": "Collect",
+        "blurb": ""
+    },
+    {
+        "uuid": "daf92b42-f82f-4a9f-a709-a39e74843891",
+        "slug": "concept",
+        "name": "Concept",
+        "blurb": ""
+    },
+    {
+        "uuid": "4e5ca742-379c-49da-9f57-54a740f3e632",
+        "slug": "conditionals",
+        "name": "Conditionals",
+        "blurb": ""
+    },
+    {
+        "uuid": "44644a80-8b1f-4a52-a999-97a3f5f9db6b",
+        "slug": "const-and-static",
+        "name": "Const and static",
+        "blurb": ""
+    },
+    {
+        "uuid": "992f5e5d-72a8-4f52-95e1-3bbe6c4f4689",
+        "slug": "count-and-sum",
+        "name": "Count and sum",
+        "blurb": ""
+    },
+    {
+        "uuid": "f060e364-b6ea-4699-bb7a-e64807a8f0dd",
+        "slug": "deref-coercion",
+        "name": "Deref coercion",
+        "blurb": ""
+    },
+    {
+        "uuid": "9cdc0b7e-520b-4d08-854b-ca7063c2dc97",
+        "slug": "derive",
+        "name": "Derive",
+        "blurb": ""
+    },
+    {
+        "uuid": "053cdc50-ca0b-4267-97f7-2d5ca4436df0",
+        "slug": "designing-custom-traits",
+        "name": "Designing custom traits",
+        "blurb": ""
+    },
+    {
+        "uuid": "c554fa18-7aba-4e96-b4bb-fc9b547252d2",
+        "slug": "enums",
+        "name": "Enums",
+        "blurb": ""
+    },
+    {
+        "uuid": "56454976-5e1b-4560-bff4-51f996534a08",
+        "slug": "explicit-return",
+        "name": "Explicit return",
+        "blurb": ""
+    },
+    {
+        "uuid": "6ba4b428-3d56-4798-ada1-79675bd5be1d",
+        "slug": "external-crates",
+        "name": "External crates",
+        "blurb": ""
+    },
+    {
+        "uuid": "e54494dc-15af-4904-b554-c583f511dca3",
+        "slug": "external-traits-as-bounds",
+        "name": "External traits as bounds",
+        "blurb": ""
+    },
+    {
+        "uuid": "3fe73200-fe19-4492-98ec-40bc17760cdd",
+        "slug": "fold",
+        "name": "Fold",
+        "blurb": ""
+    },
+    {
+        "uuid": "445f2f63-e5fb-4500-93b3-19afac753bfa",
+        "slug": "format-macro",
+        "name": "Format macro",
+        "blurb": ""
+    },
+    {
+        "uuid": "44d3d261-443d-4bbc-834e-5d35aca0cab8",
+        "slug": "from-into-iterator",
+        "name": "From into iterator",
+        "blurb": ""
+    },
+    {
+        "uuid": "cd77f4c9-3c07-4882-b828-ac99748415e5",
+        "slug": "functions",
+        "name": "Functions",
+        "blurb": ""
+    },
+    {
+        "uuid": "3848f069-afe9-4882-95ba-4112b565bdab",
+        "slug": "functions-anonymous",
+        "name": "Functions anonymous",
+        "blurb": ""
+    },
```

Comment:
```suggestion
```

Anonymous functions are only possible via closures in Rust, so there's no point in having distinct concepts.